res/css/structures/_SpacePanel.scss

```diff
@@ -327,7 +331,6 @@ $activeBorderColor: $primary-content;
             display: block;
         }
     }
-
```

Comment:
Here likewise

Reply:
the style linter is saying otherwise, it's not allowing me pass the test cases that is why

Reply:
![style](https://user-images.githubusercontent.com/38439166/165490181-a54ea8bf-a017-4e29-9d7d-e5ff71e0d036.PNG)

Reply:
That just means that it's detecting trailing spaces on that line. If you add a newline without any spaces, it should pass the linter.

Reply:
Ah, I see you did that